apps/react/src/challenges/transfer-list/Components/RightList/RightList.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import ListItems from "../ListItems/ListItems";
+
+const RightList = ({data,handleData}) => {
+  
+  return (
+    <div className="rightList-container ">
+      <ListItems data={data} handleData={handleData} direction='right'/>
+    </div>
+  );
+};
```

Comment:
can we not use a single component for LeftList and RightList

Reply:
Done